src/databricks/labs/ucx/source_code/files.py

```diff
@@ -44,12 +48,79 @@ def __repr__(self):
         return f"<LocalFile {self._path}>"
 
 
+class LocalDirectory(SourceContainer):
```

Comment:
```suggestion
class Folder(SourceContainer):
```
i think it's already clear the folder is local.

Reply:
done